recipes/ntstat/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "1.0.0" %}
+
+package:
+  name: ntstat
+  version: {{ version }}
+
+source:
+  url: https://github.com/bcgsc/ntStat/releases/download/v{{ version }}/ntStat-{{ version }}.tar.gz
+  sha256: cbdf691033727466aaa774c5b9a7fc549448280b53f59579d1a43d9e943f9abd
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('ntstat', max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - meson
+    - meson-python
+    - cmake
+    - python
```

Comment:
is this really needed here?

Reply:
Adding `python` to the build requirements fixed a version mismatch issue where my Python modules compiled from C++ were being copied to `python3.13/site-packages` during build and imported from `python3.10` during run (and test). I could not find an alternative fix.